src/components/Dialog/AlertDialog.js

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
```

Comment:
Do we need this import statement?

Reply:
Since we are using React 17 and higher we do not require this import statement